docs/cunumeric/source/user/usage.rst

```diff
@@ -41,6 +41,13 @@ using the ``LEGATE_CONFIG`` environment variable:
 
 See the :ref:`config` section :ref:`config_legate` for more information.
 
+Additionally, any Legion and Realm arguments may also be passed via the
+``LEGION_DEFAULT_ARGS`` environment variable:
+
+.. code-block:: sh
+
+  LEGION_DEFAULT_ARGS="-lg:local 1 -ll:cpu 4" legate script.py <script options>
```

Comment:
Let's advertise some more benign options (these particular flags conflict with that legate passes), here and below:
```
LEGION_DEFAULT_ARGS="-lg:sched 100 -ll:show_rsrv"
```

Reply:
https://github.com/nv-legate/cunumeric/pull/939/commits/29e7e8f93fb04357330ab2fcbfaf8ee2f5524a60